### PR TITLE
make DataSourceProtocol.item(atIndexPath: ) public

### PR DESCRIPTION
**Pull request by jessesquires**
lol whoops. 😳  This was intended to be `public`.

- Make extension method public
- Update changelog for 6.1.0
- Update changelog milestone links for new GH URLs

### CODE REVIEW

File: Source/DataSource.swift [hunk 1]
```diff
@@ -266,3 +266,11 @@ extension FetchedResultsController: DataSourceProtocol {
         return nil
     }
 }
+
+extension FetchedResultsController {
+
+    /// :nodoc:
+    public func item(atIndexPath indexPath: IndexPath) -> Item? {
+        return object(at: indexPath) as? T
```
Comment (dcaunt) [inline Source/DataSource.swift]: Nit – should be `as? Item` for consistency?

**Review by dcaunt — APPROVED** [review]
Good catch! Just a minor note.

File: Source/DataSource.swift (hunk 1 above)
Comment (jessesquires) [reply Source/DataSource.swift]: lol yep!

**Comment by dcaunt** [comment]
:shipit:

**Comment by jessesquires** [comment]
i guess i should write a couple of tests, too 😅

**Comment by jessesquires** [comment]
thanks @dcaunt ! 💯

**Pull request merged**